test/screenshotter/ss_data.yaml

```diff
@@ -16,8 +16,11 @@
 Accents: \vec{A}\vec{x}\vec x^2\vec{x}_2^2\vec{A}^2\vec{xA}^2
 AccentsText: |
     \begin{array}{l}
-    \text{\H{e}} & \text{\'\i} \\
-    \text{\H{X}} & \text{\"\i} \\
+    \text{\'\i} & \text{\.\i} & \text{\`\i} & \text{\"\i} & \text{\H\i} & \text{\r\i} \\
+    \text{\'\j} & \text{\.\j} & \text{\`\j} & \text{\"\j} & \text{\H\j} & \text{\r\j} \\
+    \text{\'a} & \text{\.a} & \text{\`a} & \text{\"a} & \text{\H{a}} & \text{\r{a}} \\
+    \text{\'A} & \text{\.A} & \text{\`A} & \text{\"A} & \text{\H{A}} & \text{\r{A}} \\
+    \text{\.I} & \text{\H e} & \text{\i ı}
```

Comment:
@davidflanagan would you like me to change `\text{\H e}` to `\text{\H ee̋}`?

Reply:
That would be great, if you don't mind. And maybe add an explicit dotted capital I as well to make the whole row the same. Thanks!